e2e/Dockerfile.opensuse

```
@@ -0,0 +1,68 @@
+        ca-certificates \
+        tar \
+        gzip \
+        nodejs-default \
+        npm-default \
+    && zypper clean --all
+
+# ---------------------------------------------------------------------------
+# Install Go (match go.mod version)
+# ---------------------------------------------------------------------------
+ARG GO_VERSION=1.25.10
+RUN curl -fsSL "https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz" \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)